.github/workflows/build.yaml

```diff
@@ -375,3 +375,38 @@ jobs:
 
       - name: Run codespell
         run: codespell -q 7 -S ".git,build*,./src/third_party/*" -I misc/codespell-allowlist.txt
+
+  deploy:
```

Comment:
I would suggest to move the deployment to a dedicated file since `build.yaml` is provided upstream but these are fork-only changes. It also keeps upstream merging cleanly.